test/utils.cpp

```diff
@@ -199,6 +242,15 @@ TEST_CASE("run_program()", "[utils]")
 	REQUIRE(utils::run_program(argv, "") == "hello world");
 }
 
+TEST_CASE("resolve_tilde()", "[utils]")
```

Comment:
Please expand this test to cover [this](https://coveralls.io/builds/20138514/source?filename=src%2Futils.cpp#L597) and [this}(https://coveralls.io/builds/20138514/source?filename=src%2Futils.cpp#L612) lines, if you can.